frontend/vue/main.ts

```diff
@@ -73,7 +74,8 @@ const components: any = {
   AmplitudeAdditionDiskWithValues,
   AmplitudeAdditionWidget,
   StatevectorAmplitudeEncoding,
-  StatevectorBinaryEncoding
+  StatevectorBinaryEncoding,
+  EigenvectorWidget
```

Comment:
It would be nicer to keep ordering them alphabetically. `EigenvectorWidget` should come after `AmplitudeAdditionWidget`.

The same applies for the import.